tests/Microsoft.Identity.Web.Test/AuthorityHelpersTests.cs

```diff
@@ -118,5 +118,207 @@ public void EnsureAuthorityIsV2(string initialAuthority, string expectedAuthorit
             options.Authority = AuthorityHelpers.EnsureAuthorityIsV2(options.Authority);
             Assert.Equal(expectedAuthority, options.Authority);
         }
+
+        [Theory]
+        [MemberData(nameof(AddAuthorityQueryToOptionsTheoryData))]
+        public void AddAuthorityQueryToOptions(AuthorityHelpersTheoryData theoryData)
+        {
+            // arrange
+            MicrosoftIdentityOptions options = new MicrosoftIdentityOptions
+            {
+                Authority = theoryData.Authority,
+            };
+
+            // act
+            AuthorityHelpers.AddAuthorityQueryToOptions(options);
+
+            // assert
+            Assert.NotNull(options.ExtraQueryParameters);
+            Assert.Equal(options.ExtraQueryParameters.Count, theoryData.ExpectedExtraQueryParameters.Count);
```

Comment:
this is switched, expected is supposed to be the first argument

Reply:
Switched!